R/dummy_functions.R

```diff
@@ -88,21 +89,31 @@ example_datasets <- function() { # nolint
 #'   shinyApp(app$ui, app$server)
 #' }
 #' @export
-example_module <- function(label = "example teal module", datanames = "all") {
```

Comment:
why?

Reply:
Example didn't have show-r-code before so vignettes explaining how to make app reproducible with `teal_data |> within()` etc. were not very helpful. Would you hardcode `src` inside instead?

Reply:
Yes - I doubt this argument will be used at all. This function is for intended for internal use only.